challenge.rb

```diff
@@ -1,26 +1,34 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.collect do |word|
+    word.capitalize
+  end
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select do |animal|
+    animal == "dog"
+  end
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject do |animal|
```

Comment:
nice. in most cases, we'd write this `input.reject { |animal| animal == "dog" }`. same with `.select` above.